src/xenia/base/bit_stream.cc

```diff
@@ -111,6 +111,8 @@ size_t BitStream::Copy(uint8_t* dest_buffer, size_t num_bits) {
   // First: Copy the first few bits up to a byte boundary.
   if (rel_offset_bits) {
     uint64_t bits = Peek(8 - rel_offset_bits);
+    uint8_t clear_mask = ~((1 << rel_offset_bits) - 1);
```

Comment:
I'd suggest explicit `uint8_t(1)` usage instead of `1`, especially if `~` is involved, to more clearly specify the sign and the amount of bits (also on line 137).